setup.cfg

```
@@ -90,41 +90,41 @@ dirac =
 console_scripts =
     # AccountingSystem
     dirac-accounting-decode-fileid = DIRAC.AccountingSystem.scripts.dirac_accounting_decode_fileid:main
-    dirac-admin-accounting-cli = DIRAC.AccountingSystem.scripts.dirac_admin_accounting_cli:main
+    dirac-admin-accounting-cli = DIRAC.AccountingSystem.scripts.dirac_admin_accounting_cli:main [admin]
     # ConfigurationSystem
-    dirac-admin-add-resources = DIRAC.ConfigurationSystem.scripts.dirac_admin_add_resources:main
-    dirac-admin-add-shifter = DIRAC.ConfigurationSystem.scripts.dirac_admin_add_shifter:main
-    dirac-admin-add-site = DIRAC.ConfigurationSystem.scripts.dirac_admin_add_site:main
-    dirac-admin-bdii-info = DIRAC.ConfigurationSystem.scripts.dirac_admin_bdii_info:main
-    dirac-admin-check-config-options = DIRAC.ConfigurationSystem.scripts.dirac_admin_check_config_options:main
-    dirac-admin-sort-cs-sites = DIRAC.ConfigurationSystem.scripts.dirac_admin_sort_cs_sites:main
-    dirac-admin-voms-sync = DIRAC.ConfigurationSystem.scripts.dirac_admin_voms_sync:main
-    dirac-configuration-cli = DIRAC.ConfigurationSystem.scripts.dirac_configuration_cli:main
-    dirac-configuration-dump-local-cache = DIRAC.ConfigurationSystem.scripts.dirac_configuration_dump_local_cache:main
-    dirac-configuration-shell = DIRAC.ConfigurationSystem.scripts.dirac_configuration_shell:main
+    dirac-admin-add-resources = DIRAC.ConfigurationSystem.scripts.dirac_admin_add_resources:main [admin]
+    dirac-admin-add-shifter = DIRAC.ConfigurationSystem.scripts.dirac_admin_add_shifter:main [admin]
+    dirac-admin-add-site = DIRAC.ConfigurationSystem.scripts.dirac_admin_add_site:main [admin]
+    dirac-admin-bdii-info = DIRAC.ConfigurationSystem.scripts.dirac_admin_bdii_info:main [admin]
+    dirac-admin-check-config-options = DIRAC.ConfigurationSystem.scripts.dirac_admin_check_config_options:main [admin]
+    dirac-admin-sort-cs-sites = DIRAC.ConfigurationSystem.scripts.dirac_admin_sort_cs_sites:main [admin]
+    dirac-admin-voms-sync = DIRAC.ConfigurationSystem.scripts.dirac_admin_voms_sync:main [admin]
+    dirac-configuration-cli = DIRAC.ConfigurationSystem.scripts.dirac_configuration_cli:main [admin]
+    dirac-configuration-dump-local-cache = DIRAC.ConfigurationSystem.scripts.dirac_configuration_dump_local_cache:main [admin]
+    dirac-configuration-shell = DIRAC.ConfigurationSystem.scripts.dirac_configuration_shell:main [admin]
     # Core
-    dirac-agent = DIRAC.Core.scripts.dirac_agent:main [server]
+    dirac-agent = DIRAC.Core.scripts.dirac_agent:main [server,pilot]
     dirac-configure = DIRAC.Core.scripts.dirac_configure:main
     dirac-executor = DIRAC.Core.scripts.dirac_executor:main [server]
     dirac-externals-requirements = DIRAC.Core.scripts.dirac_externals_requirements:main
```

I think this one will become useless once we're only using DIRACOS 2? (maybe worth leaving commented so we remember to clean it up in v8)
```suggestion
    # dirac-externals-requirements = DIRAC.Core.scripts.dirac_externals_requirements:main
```